packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -145,6 +145,11 @@ export class FASTTabs extends FASTElement {
         }
     }
 
+    /**
+     * Function that is invoked whenever the selected tab or the tab collection changes.
+     *
+     * @protected
```

Comment:
This tsdoc tag indicates whether the method is in the public API or is internal, and isn't connected to the TypeScript scope.

```suggestion
     * @public
```

Reply:
done